inftrees.c

```
@@ -13,6 +13,8 @@
 #  include "arch/x86/x86_intrins.h"
 #elif defined(__ARM_NEON) || defined(__ARM_NEON__)
 #  include "arch/arm/neon_intrins.h"
+#elif defined(__ALTIVEC__)
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)